CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKImage.java

```diff
@@ -118,22 +132,25 @@ public void finalize() throws Throwable {
         super.finalize();
     }
     
-    public static boolean HasDepthComponent(int format) {
-        return  format == VK_FORMAT_D16_UNORM ||
-                format == VK_FORMAT_X8_D24_UNORM_PACK32 ||
-                format == VK_FORMAT_D32_SFLOAT  ||
-                format == VK_FORMAT_D16_UNORM_S8_UINT ||
-                format == VK_FORMAT_D24_UNORM_S8_UINT ||
-                format == VK_FORMAT_D32_SFLOAT_S8_UINT;
-    }
     
-    public static boolean HasStencilComponent(int format) {
-        return format == VK_FORMAT_S8_UINT ||
-               format == VK_FORMAT_D16_UNORM_S8_UINT ||
-               format == VK_FORMAT_D24_UNORM_S8_UINT ||
-               format == VK_FORMAT_D32_SFLOAT_S8_UINT;
+    public int Transition(int newLayout) {
```

Comment:
add comment block for funciton

Reply:
Again another one for who ever has time, image layout transitioning is not something I know intimately yet so would need a bit of research to comment correctly.